crypto/src/main/scala/org/bitcoins/crypto/NetworkElement.scala

```diff
@@ -1,6 +1,5 @@
-package org.bitcoins.core.protocol
+package org.bitcoins.crypto
```

Comment:
Maybe a better abstraction would be to keep `NetworkElement` in core.protocol and create a `Serializable` for crypto, then have `NetworkElement` extend `Serializable`